styles/styles.css

```diff
@@ -53,6 +53,11 @@
   }
 }
 
+/* global layout */
+html {
+  font-size: 62.5%
```

Comment:
Per the slack conversation in https://cq-dev.slack.com/archives/C056LDS3RJ4/p1683321923972019 I decided to take the path of least resistance for my first implementation and went with the same relative sizing implementation used in the clients original page.